adapters/Unity/UnityAdapter/GADMAdapterUnitySingleton.h

```diff
@@ -29,19 +29,17 @@
 - (void)initializeWithGameID:(NSString *)gameID;
 
 /// Requests a reward-based video ad with |adapterDelegate|.
-- (void)requestRewardedAdWithDelegate:
-    (id<GADMAdapterUnityDataProvider, UnityAdsExtendedDelegate>)adapterDelegate;
+- (void)requestRewardedAdWithDelegate:(NSString *)gameID
```

Comment:
Param doesn't match the name. Method name is requestRewardedAdWithDelegate:delegate: which is odd.

Instead of adding another param like this, you could instead add a getGameID method to the GADMAdapterUnityDataProvider protocol. This delegate object getting passed in is already providing placement ID via this data provider protocol.